common-npm-packages/webdeployment-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "azure-pipelines-tasks-webdeployment-common",
-    "version": "4.222.1",
+    "version": "4.222.2",
```

Comment:
Since we passed last Tuesday of 3rd week of sprint, we can name it as `4.223.0` (if rules are same with tasks repository)